code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1188,6 +1249,40 @@
 	..()
 	. = 1
 
+/datum/reagent/medicine/alprazolam
+	name = "Alprazolam"
+	description = "Used to supress extreme cases of mental disorders."
+	reagent_state = LIQUID
+	color = "#4a07e7"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	overdose_threshold = 15
+
+/datum/reagent/medicine/alprazolam/on_mob_metabolize(mob/living/L)
+	..()
+	ADD_TRAIT(L, TRAIT_RELAXED, PSYCH_TRAIT)
+
+/datum/reagent/medicine/alprazolam/on_mob_end_metabolize(mob/living/L)
+	REMOVE_TRAIT(L, TRAIT_RELAXED, PSYCH_TRAIT)
+	..()
+
+/datum/reagent/medicine/alprazolam/on_mob_life(mob/living/carbon/M)
+	M.jitteriness = max(0, M.jitteriness-6)
+	SEND_SIGNAL(M,COMSIG_SET_SANITY,2,SANITY_NEUTRAL)
```

Comment:
I specificaly removed this signal in the past because mood components are already not the best component with how many snowflake signals it has. Signals should not be created with a single intent and named according to what they do, they should be created describing what's happening to set off the event. The component is in charge of deciding what events to listen for and this encourages people to use the signal in more than one place which is not good.

If you want to do this in a good way make a generic signal for reagent processing so the component can listen to particular ones